src/InputParser.cpp

```diff
@@ -1 +1,99 @@
-#include "../include/InputParser.h"
+﻿//
+// Created by MateuszDorobek on 07.01.2018.
+//
+#include <algorithm>
+#include <string>
+#include "..\include\InputParser.h"
+
+void InputParser::extractSentences(std::string text) {
+	text = toLower(text);
+	size_t pos = 0;
+	size_t pos1 = 0;
+	size_t pos2 = 0;
+	size_t pos3 = 0;
+	std::string token;
+	while ((pos = text.find('\n')) != std::string::npos)
+		std::replace(text.begin(), text.end(), '\n', ' ');
+	while ((pos1 = text.find('.')) != std::string::npos ||
+		(pos2 = text.find('?')) != std::string::npos ||
+		(pos3 = text.find('!')) != std::string::npos
+		) {
+		pos1 = text.find('.');
+		pos2 = text.find('?');
+		pos3 = text.find('!');
+		pos = min(pos1, min(pos2, pos3));
+		token = text.substr(0, pos + 1);
+		push(token);
+		text.erase(0, pos + 1);
+	}
+}
+
+void InputParser::push(std::string s) {
+	while (s.at(0) == ' ')
+		s = s.substr(1, s.size());
+	if (s == "." || s == "?" || s == "!")
+		return;
+	sentences.push_back(s);
+};
+
+string InputParser::toLower(string text) {
+	std::transform(text.begin(), text.end(), text.begin(), ::tolower);
+	return text;
+}
+
+std::vector<std::string> InputParser::extractWordsFromSentence(std::string sent) {
+	sent = toLower(sent);
```

Comment:
It would look better if variable "sent" was named simply "sentence" - unnecessary shortening.